kouncil-backend/src/main/java/com/consdata/kouncil/schema/clusteraware/SchemaAwareClusterService.java

```diff
@@ -39,6 +33,24 @@ public boolean clusterHasSchemaRegistry(String serverId) {
         return schemaAwareCluster.containsKey(serverId);
     }
 
+    public void reloadSchemaConfiguration(KouncilConfiguration kouncilConfiguration){
+        schemaAwareCluster = new ConcurrentHashMap<>();
+        kouncilConfiguration.getClusterConfig().forEach((clusterKey, clusterValue) -> {
+            try {
+                SchemaRegistryClient schemaRegistryClient = clusterValue.getSchemaRegistry() != null
```

Comment:
You can simplify this few lines of code. You can just check if clusterValue.getSchemaRegistry() is null and then inside braces build client and facade